paper-ripple.html

```diff
@@ -547,6 +571,6 @@
         }
       }
     });
-  });
+  })();
```

Comment:
Style question: call the anonymous function inside the parentheses or outside?

The origin post for [IIFEs](http://benalman.com/news/2010/11/immediately-invoked-function-expression/) suggests enclosing the function call inside the parentheses, and the [Wikipedia article on IIFEs](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression) suggests the same syntax.

```js
(function() {

}());
```